src/main/java/at/gammastrahlung/monopoly_server/game/AuctionSystem.java

```diff
@@ -0,0 +1,93 @@
+package at.gammastrahlung.monopoly_server.game;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import at.gammastrahlung.monopoly_server.game.gameboard.Property;
+
+/**
+ * This class manages the auction system in a Monopoly game for Android.
```

Comment:
This does not need to be just for an android app.
The server has nothing to do with android.